### ISSUE TITLE
Battery Charge Not Maintaining Percentage After Reaching 100% | ASUS ROG ALLY Z1 EXTREME

### ISSUE THREAD

**Issue by RyzerZ** [ublue-os/bazzite#1734]
### Describe the bug

When the battery reaches 100% charge while plugged in, it stops charging but does not maintain the charge percentage. The battery percentage starts to decrease while still connected to the power source.

### What did you expect to happen?

I expected the battery to remain at 100% charge while plugged in, ensuring the device maintains its power without discharging.

**Comment by Tobe95** [comment]
Do you happen to know if the battery stops discharging at some point, for example after reaching 90%?

I think lots of electronic devices, after being fully charged, allow the battery to deplete to a certain percentage, since it's not good for battery health to constantly charge to 100%. 

Provided your Ally does not go below 85-90%, this could be intended behaviour.

**Comment by RyzerZ** [comment]
@Tobe95 I know it’s not ideal to exceed 85%, but I enjoy using it with a projector, and having it discharge is an inconvenience. I’d like it to stay at the set limit.

**Comment by antheas** [comment]
Yes, this is normal for all devices. Usually the limit is 5-10% less. Always charging the device would damage the battery. This is handled by the EC of the device, and in this case to what Asus chose.

Certain thinkpads would allow you to do what you want, even though it is very not recommended.
